Lesson08_Task01.py

```diff
@@ -0,0 +1,38 @@
+"""
+Lesson08 Task01.
+
+Given the list of integers ( positive , negative, zeros)
+
+Find max multiplication of three values in list
+l1 = [-7, 1, 2, 0, 3, 12, -24, 5, -1, 6]
+
+Input: [-1, 1, 2, 0, 3, 12, 4, 5, -1, 6]
+Output: Max value = "360". Nums are: (12, 5, 6)
+
+Input: [-7, 1, 2, 0, 3, 12, -24, 5, -1, 6]
+Output: Max value = "2016". Nums are: (-7, 12, -24)
+"""
+
+from itertools import permutations
+
+l1 = [-7, 1, 2, 0, 3, 12, -24, 5, -1, 6]
+
+copy_l1 = l1[:]
+
+# remove all 0s from copy_l1
```

Comment:
Unfortunately solution is incorrect.
Please try you algorithm vs l1 = [-1, 0, 0, 0, 1, 2]  it gives Max value = "-2". Nums are: (-1, 1, 2)
But expected Max value = "0". Nums are: (0, 1, 2)

In addition for the next time please do NOT use itertools  )) I have shared with group solution already during Lesson_09 and group was notified that such solution we NOT use.
https://github.com/antohaUa/hillel_aqa_050623/blob/main/Lesson_09/00_prev_lesson.py#L28

Advice for the next time try to sort list and find multiplication max patterns. Or find your own algorithm )

Reply:
fixed by using another solution